index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <!--Basic page Needs
+  //____________________________________-->
+  <meta charset="utf-8">
+  <meta name="description" content=" Protfolio site">
+  <meta name="author" content="Mariam Bederu">
+
+  <!-- Mobile Devices
+  ______________________________________-->
+
+
+  <!--FONT
+  ______________________________________-->
+  <link href="http://fonts.googleapis.com/css?family=Poiret+One" rel="stylesheet" type="text/css">
+  <link href="http://fonts.googleapis.com/css?family=Questrial" rel="stylesheet" type="text/css">
+  <link href="http://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+  <!--CSS
+  ______________________________________-->
+  <link rel="stylesheet" href="css/skeleton.css">
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="fonts/font-awesome/css/font-awesome.min.css">
+<!--JAVASCRIPT
+  ______________________________________-->
+<!--<script src="index.js"></script>-->
+
+<title>Mariam Bederu</title>
+</head>
+
+<!--BODY
+  ______________________________________-->
+
+<body>
+
+  <header class="nav blue">
+    <div class="container">
+      <div class="twelve columns">
+        <h1 class="logo">Mariam Bederu</h1>
+      <div id="cssmenu">
```

Comment:
From this name, I'm not sure I would understand what cssmenu means. A little vauge.